_config.yml

```diff
@@ -0,0 +1,9 @@
+# Note: This file is used only for redirecting visitors from the old ScalarDB docs site hosted in this repository to the new ScalarDB docs site. This file can be deleted after the old ScalarDB docs site hosted in this repository is no longer deployed.
+google_analytics: "G-Q4TKS77KCP"
+
+defaults:
+  - scope:
+      path: "docs" # Specifies where the docs are located.
+      type: "default" # Defines the type for docs.
+    values:
+      layout: default # Defines the template type used for docs (templates are located in the "_layouts" folder). 
```

Comment:
This mark means "No newline at end of file." It's recommended to add a newline at the end of file https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Thank you for catching that! I've added a newline and mentioned it in the style guide I am working on.